src/main/java/com/wow/server/data/repository/UserRepository.java

```diff
@@ -1,11 +1,17 @@
 package com.wow.server.data.repository;
 
+import com.wow.server.data.model.User;
+import com.wow.server.data.projection.UserMinimalProjection;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
-import com.wow.server.data.model.User;
+import java.util.List;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, Long>{
 
+    @Query("select new com.wow.server.data.projection.UserMinimalProjection(u.userId, u.displayName) from User u")
```

Comment:
I think it should be like top X sort by Y
not sure what Y might be... date joined? or other field?

But then again do wea want ALL users, or TOP X WHERE Y SORT Z ?
like top 50 performewrs
top 50 people with nmost subscriptions 
etc

Reply:
I totally, agree at the point where I'm "not sure", because it was not in task requirements. Probably, I'll just extract it into separate PR where we can discuss it. Hopefully with somebody from FE and "business" to understand their needs here. It was just some kind of forecast in needs.